handle_rentals.rb

```diff
@@ -0,0 +1,43 @@
+require './rental'
+
+class HandleRentals
+  def initialize(people, books)
+    @rentals = []
+    @handle_people = people
+    @handle_books = books
+  end
+
+  def create_rental
+    puts "Select book from this books using the number: \n"
+    @handle_books.list_of_books
+    book_number = gets.chomp
+    @handle_books.books.each_with_index { |book, index| @selected_book = book if (index + 1) == book_number.to_i }
+    puts "Select a person fome the following list of members buy id:\n"
```

Comment:
- [ ] There is some typo here. Instead of `buy`, it should be `by` and instead of `fome`, it should be `from`. Please fix that